oauth2client/contrib/gce.py

```diff
@@ -55,8 +52,7 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
```

Comment:
On that note, did we require these to be refreshed before usage before? Can you verify that simple calls with google api python client still work as expected with this?